pandas/tslib.pyx

```diff
@@ -225,8 +225,12 @@ class Timestamp(_Timestamp):
     for the entries that make up a DatetimeIndex, and other timeseries
     oriented data structures in pandas.
 
-    Parameters
-    ----------
+    There are essentially three calling conventions for the constructor. The
+    first, legacy form accepts four parameters. They can be passed by
+    position or keyword.
+
+    Legacy Parameters
```

Comment:
don't call these ``Legacy`` Parameters; these are in fact the normal calling convention. The ``datetime.datetime`` convention are the compat parameters (but you don't have to call it differently).